app/views/posts/show.html.erb

```diff
@@ -1,31 +1,40 @@
 <div id="show">
- <div class="posts">
-            <% @posts.each_with_index do |post, ind| %>
+ <div class="posts comments">
                        <div class="post-head">
-                            <b>Post#<%= ind+1 %> by <%=@user.name%></b> 
+                            <b>Post#<%= 1 %> by <%=@user.name%></b> 
                             <div class="like-comment">
-                                <p>Comments : <%= post.comments_counter %>,  </p>
-                                <p>Likes : <%= post.likes_counter %></p>
+                                <p>Comments : <%= @post.comments_counter %>,  </p>
+
+                                <%= form_with url:"/users/#{@post.author.id}/posts/#{@post.id}/add_like"  do |form| %>
+                                    <%= form.submit '🖤'%>
+                                <% end %>
+
+                                <p>Likes : <%= @post.likes_counter %></p>
+
+                                <%= form_with url:"/users/#{@post.author.id}/posts/#{@post.id}/add_comment" , method: :post, scope: :comment  do |form| %>
+                                    <%= form.text_area :text %>
+                                    <%= form.submit 'Send'%>
+                                <% end %>
```

Comment:
- [ ] [optional] I would recommend you move the comment form and add like to the bottom f the page below the comments this way your page will look better. Also would be better if your comments appeared in a list instead of next to each other, you can use <ul> <li> to achieve this.